homeassistant/components/backup/agent.py

```diff
@@ -8,10 +8,21 @@
 from typing import Any, Protocol
 
 from homeassistant.core import HomeAssistant
+from homeassistant.exceptions import HomeAssistantError
 
 from .models import BackupUploadMetadata, BaseBackup
 
 
+class BackupAgentError(HomeAssistantError):
+    """Base class for backup agent errors."""
+
+
+class BackupAgentUnreachableError(BackupAgentError):
+    """Raised when the agent can't reach it's API."""
```

Comment:
```suggestion
    """Raised when the agent can't reach its API."""
```

Reply:
Fixed in https://github.com/home-assistant/core/pull/130890